pkg/controller/datavolume-controller.go

```diff
@@ -492,6 +504,18 @@ func (r *DatavolumeReconciler) reconcileClone(log logr.Logger,
 	prePopulated bool,
 	selectedCloneStrategy cloneStrategy) (reconcile.Result, error) {
 
+	// When cloning a PVC, if the target's size is not specified,
+	// said value can be attainable from the source PVC
+	targetRequest, hasTargetRequest := pvcSpec.Resources.Requests[corev1.ResourceStorage]
+	if hasTargetRequest && targetRequest.IsZero() {
```

Comment:
Yeah, after `RenderPvcSpec` it should be impossible to have a false `hasTargetRequest`. However, I preferred to check the value before accessing `IsZero()` just to be extra sure to avoid possible segfaults. I can add the `!hasTargetRequest || ...` if you prefer.